Tasks/SonarQubePostTest/SonarQubeMetrics.ps1

```diff
@@ -1,3 +1,6 @@
+$script:qualityGateStatus = ""
```

Comment:
I'm replacing task variables with script scoped variables. These are persisted for the duration of this script. Task variables use process env variables underneath, however the new core clr agent uses a separate process to run each task so setting and getting a task variable withing the same task no longer works.